.drone.yml

```diff
@@ -13,15 +13,17 @@ steps:
     settings:
       mode: lint
       chart: ./helm
-      values_file: ./helm/values.yaml
+      values_files:
+        - ./helm/values.yaml
```

Comment:
Should we rename this file to `dev-values`? Maybe they want to be in a new `values/…` directory as well so they’re not spread out all over the project root 🤔